src/GitVersionCore/Model/FileLock.cs

```diff
@@ -0,0 +1,25 @@
+using GitVersion.FileLocking;
+using GitVersion.FileLocking.Abstractions;
+using System;
+using System.IO;
+
+namespace GitVersion.Helpers
```

Comment:
[Please avoid using names such as `Helpers`](https://softwareengineering.stackexchange.com/a/247277/20746).

```suggestion
namespace GitVersion
```